test/flycheck-test.el

```diff
@@ -3473,6 +3473,14 @@ See https://github.com/flycheck/flycheck/issues/531 and Emacs bug #19206"))
      '(9 9 warning "Ignored `error` returned from `os.Stat(\"enoent\")`"
          :checker go-errcheck))))
 
+(flycheck-ert-def-checker-test go-errcheck go nil
```

Comment:
@dominikh The name must be different if there are to be multiple tests for the same checker.  `nil` simply means "default" here.